### PR TITLE
Added docker build env for fedora25

### PR DESCRIPTION
**Pull request by petrkotas**
I have added support for Fedora25, since I needed it.
If it fits Your plan, would You kindly merge it? I will be happy to maintain further versions.

Best regards,
Petr

---
This change is [Reviewable](https://reviewable.io/reviews/dhiltgen/docker-machine-kvm/50)

### CODE REVIEW

**Comment by gbraad** [comment]
The CentOS binary would also work, but can you update to Fedora 26?

File: Dockerfile.fedora25 [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FROM fedora:25
```
Comment (gbraad) [inline Dockerfile.fedora25:1]: `fedora:26` now

**Review by gbraad — APPROVED** [review]
although based on fedora:25, this is still usable

File: Dockerfile.fedora25 (hunk 1 above)
Comment (petrkotas) [reply Dockerfile.fedora25:1]: Sorry for the delay. Yes I will change it for the fedora 26. Sorry for the late reply.

**Comment by petrkotas** [comment]
Actually, I have tested the CentOS and my build. They both work the same. But this is a more obvious approach to broader public :).

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/dhiltgen/docker-machine-kvm/50#-:-Ks_84XQa8wtYMz5zXpT:b-109nmn)*

**Comment by petrkotas** [comment]
Is this PR still relevant? Or can we close it?

**Comment by gbraad** [comment]
If the project gets transfered to the docker-machines organization we can
have a new look at this =>
https://github.com/dhiltgen/docker-machine-kvm/issues/67

**Comment by petrkotas** [comment]
I am closing this PR. It is no longer relevant.

**Pull request closed without merging**